patches/resin-cli-visuals+1.4.3.patch

```diff
@@ -0,0 +1,22 @@
+diff --git a/node_modules/resin-cli-visuals/build/widgets/drive/index.js b/node_modules/resin-cli-visuals/build/widgets/drive/index.js
+index 475c0bc..338fc3c 100644
+--- a/node_modules/resin-cli-visuals/build/widgets/drive/index.js
++++ b/node_modules/resin-cli-visuals/build/widgets/drive/index.js
+@@ -21,7 +21,7 @@ chalk = require('chalk');
+ 
+ Promise = require('bluebird');
+ 
+-drivelist = Promise.promisifyAll(require('drivelist'));
++drivelist = require('drivelist');
```

Comment:
So I found (the hard way) that `resin-cli-visuals` was broken (following an update to the npm `drivelist` package), preventing `os intialize` from working. I intend to submit a PR (simple enough) in the `resin-cli-visuals` repo, but for interim testing I found it convenient to have this patch.

Reply:
Yeah I'd definitely prefer we upstream this before merging of this PR.

Reply:
Upstream merged, PR updated to use `resin-cli-visuals` v1.4.4.